### DISCUSSION TITLE
How to access MAC WebCrawler Connector's in Githup URL

### DISCUSSION THREAD

**Discussion by tiru** [MuleSoft-AI-Chain-Project/mule-ai-chain-connector#65] [Q&A]
Hello Team,

Hope all are doing great!

I am not able to get gitup URL for webcrawler connector's  from https://mac-project.ai/docs/mac-webcrawler/getting-started

Can you please help me, How to get MAC WebCrawler Connector's Gitup URL?

**Comment by amirkhan-ak-sf** [comment]
Hi @tiru, Webcrawler should be available now: https://github.com/MuleSoft-AI-Chain-Project/mac-web-crawler

we will need to update the docs.
